features/backend/shipping_categories.feature

```diff
@@ -60,6 +62,18 @@ Feature: Shipping categories
          Then I should be on the shipping category index page
           And I should see "Shipping category has been successfully updated."
 
+    Scenario: Cannot update shipping category code
+         When I am editing shipping category "Heavy"
+         Then The code field should be disable
+
+    Scenario: Try add shipping category with existing code
+        Given I am on the shipping category creation page
+         When I fill in "Name" with "Middle"
+         When I fill in "Code" with "SC1"
+          And I press "Create"
+         Then I should still be on the shipping category creation page
+          And I should see "The shipping category with given code already exists"
```

Comment:
I know this file has all steps aligned, but we decided to not do this, so for new scenarios please don't align them to "I". We will change it everywhere later. :)